core/cautils/mappingnode.go

```diff
@@ -0,0 +1,45 @@
+package cautils
+
+type ObjectID struct {
+	apiVersion string
+	kind       string
+}
+
+type MappingNode struct {
+	ObjectID           *ObjectID
+	Field              string
+	Value              string
+	TemplateFileName   string
+	TemplateLineNumber int
+}
+
+type MappingNodes struct {
+	Nodes            []MappingNode //Map line number of chart to template obj
+	TemplateFileName string
+}
+
+type FileMapping struct {
+	Mapping map[string]*MappingNodes
```

Comment:
Any specific reason for declaring it as `map[string]*MappingNodes` and not `map[string]MappingNodes`?

Reply:
If we are not planning on extending the FileMapping I would go with aliasing:
`type FileMapping map[string]*MappingNodes`

Reply:
Fix